kitsune/llm/questions/classifiers.py

```diff
@@ -0,0 +1,71 @@
+from typing import TYPE_CHECKING, Any
+
+from django.db import models
+from langchain.schema.runnable import RunnableLambda, RunnablePassthrough
+
+from kitsune.llm.questions.prompt import spam_parser, spam_prompt, topic_parser, topic_prompt
+from kitsune.llm.utils import get_llm
+from kitsune.products.utils import get_taxonomy
+
+DEFAULT_LLM_MODEL = "gemini-2.5-flash-preview-04-17"
+HIGH_CONFIDENCE_THRESHOLD = 75
+LOW_CONFIDENCE_THRESHOLD = 60
+
+if TYPE_CHECKING:
+    from kitsune.questions.models import Question
+
+
+class ModerationAction(models.TextChoices):
+    NOT_SPAM = "not_spam", "Not Spam"
+    SPAM = "spam", "Spam"
+    FLAG_REVIEW = "flag_review", "Flag for Review"
+
+
+def classify_question(question: "Question") -> dict[str, Any]:
+    """
+    Analyze a question for spam and, if not spam or low confidence, classify the topic.
+    Returns a dict with keys: action, spam_result, topic_result (optional).
+    """
+    llm = get_llm(model_name=DEFAULT_LLM_MODEL)
```

Comment:
I don't think a non-zero `temperature`setting makes sense for this use case. We want as much consistency as possible.